### ISSUE TITLE
Should we use btrfs for the game partition?

### ISSUE THREAD

**Issue by Killea** [rg35xx-cfw/rg35xx-cfw.github.io#53]
I'm asking because I saw drivers for btrfs in the tools folder.😊

**Comment by scottchiefbaker** [comment]
I don't think btrfs gets us anything that a simple ext4/fat32 partition wouldn't. It's certainly less supported. I would not vote to change to btrfs.

**Comment by acmeplus** [comment]
Yeah, btrfs is not supported with this kernel. It's included in the tools because batocera by default includes those tools. Sorry for the confusion.
